### PR TITLE
archinstall: add Dutch translation

### PR DESCRIPTION
**Pull request by leonvsc**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.nl/linux/archinstall.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# archinstall
+
+> Guided Arch Linux installer with a twist.
+> More information: <https://archinstall.readthedocs.io>.
```
Comment (navarroaxel) [inline pages.nl/linux/archinstall.md:4]:
```suggestion
> Meer informatie: <https://archinstall.readthedocs.io>.
```

File: pages.nl/linux/archinstall.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# archinstall
+
+> Guided Arch Linux installer with a twist.
```
Comment (navarroaxel) [inline pages.nl/linux/archinstall.md:3]: Are you sure this is well translated? :eyes:

File: pages.nl/linux/archinstall.md (hunk 1 above)
Comment (leonvsc) [reply pages.nl/linux/archinstall.md]: Fixed this one.

File: pages.nl/linux/archinstall.md (hunk 2 above)
Comment (leonvsc) [reply pages.nl/linux/archinstall.md]: Forgot the whole header. Whoopsie.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**